.travis.yml

```diff
@@ -1,16 +1,47 @@
 language: python
-python: 2.7
+python: 
+  - 2.7
+  - 3.5
 addons:
   firefox: 47.0.1
-env:
-- TOXENV=tests DISPLAY=:99.0
-- TOXENV=flake8
+matrix:
+  include:
+    - python: 2.7
+      env: 
+        - TOXENV=flake8
+    - python: 2.7
+      env: 
+        - DISPLAY=:99.0
+        - TOXENV=py27
+    - python: 2.7
+      env: 
+        - DISPLAY=:99.0
+        - TOXENV=py27-marionette
+    - python: 3.5
+      env: TOXENV=flake8
+    - python: 3.5
+      env: 
+        - DISPLAY=:99.0
+        - TOXENV=py35
+    - python: 3.5
+      env: 
+        - DISPLAY=:99.0
+        - TOXENV=py35-marionette
+  exclude:
+    - python: 2.7
+    - python: 3.5
```

Comment:
Not if you're using tox-travis. No evironment variable is fine, tox-travis detects the version of python in use and selects the appropriate tox environment. For example on python 3.5 it will find and run `py35`.